test/stablecoin_converter.js

```diff
@@ -354,45 +356,45 @@ contract("StablecoinConverter", async (accounts) => {
       await feeToken.givenAnyReturnBool(true)
       await erc20_2.givenAnyReturnBool(true)
 
-      await stablecoinConverter.deposit(feeToken.address, 10000, { from: user_1 })
-      await stablecoinConverter.deposit(erc20_2.address, 10000, { from: user_3 })
+      await stablecoinConverter.deposit(feeToken.address, feeAdded(10000), { from: user_1 })
+      await stablecoinConverter.deposit(erc20_2.address, feeAdded(10000), { from: user_3 })
       await stablecoinConverter.deposit(erc20_2.address, 2000, { from: user_2 }) // needed to pay fees
 
       await stablecoinConverter.addToken(erc20_2.address)
 
       const batchIndex = (await stablecoinConverter.getCurrentBatchId.call()).toNumber()
 
-      const orderId1 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 1, 0, true, batchIndex + 1, feeSubtracted(10000) - 1, 10000, { from: user_1 })
-      const orderId2 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 0, 1, true, batchIndex + 1, feeSubtracted(10000) - 1, 10000, { from: user_2 })
-      const orderId3 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 1, 0, true, batchIndex + 1, feeSubtracted(10000) - 1, 10000, { from: user_2 })
-      const orderId4 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 0, 1, true, batchIndex + 1, feeSubtracted(10000) - 1, 10000, { from: user_3 })
+      const orderId1 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 1, 0, true, batchIndex + 1, 10000, feeAdded(10000), { from: user_1 })
+      const orderId2 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 0, 1, true, batchIndex + 1, 9990, feeAdded(9990), { from: user_2 })
+      const orderId3 = await sendTxAndGetReturnValue(stablecoinConverter.placeOrder, 1, 0, true, batchIndex + 1, 9981, feeAdded(9981), { from: user_2 })
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FhoyBmp5ccZjGUCmmM5%2Fgiphy.gif&token=complicated" data-gitmeme-token="complicated"><img src="https://media3.giphy.com/media/hoyBmp5ccZjGUCmmM5/giphy.gif" title="Created by gitme.me with /complicated"/></a>

I don't see through these test cases and magic number. I hope @bh2smith does, otherwise we should probably look into simplifying them going forward.

Reply:
I am also not sure I am confident in the meaning of these numbers... Perhaps a comment describing where they came from could help.